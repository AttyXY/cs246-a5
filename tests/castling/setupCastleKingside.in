setup
+ K e1
+ R h1
+ k e8
+ r h8
done
game human human
move e1 g1
move e8 g8
