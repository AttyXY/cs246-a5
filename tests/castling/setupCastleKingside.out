WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

WELCOME TO SETUP MODE!
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- + [PieceType] [Coordinate] (to add piece)
- - [Coordinate] (to remove piece)
- = [Colour] (to set turn)
- done (to exit setup mode)

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--R
   abcdefgh

8  ----k---
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--R
   abcdefgh

8  ----k--r
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--R
   abcdefgh

PLEASE ENTER THE FOLLOWING COMMANDS TO BEGIN:
- game [player1] [player2]

STARTING GAME!

8  ----k--r
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--R
   abcdefgh

8  ----k--r
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -----RK-
   abcdefgh

8  -----rk-
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -----RK-
   abcdefgh


PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- setup (to create new setup)
- game (to continue with previous setup)
- quit (to exit)

THANKS FOR PLAYING CHESS!
Final Score:
White: 0
Black: 0
